src/blob.c

```diff
@@ -369,6 +399,11 @@ blob_slice(
 	int		exclusive,
 	typval_T	*rettv)
 {
+    // Inicializar rettv primero
+    clear_tv(rettv);
```

Comment:
The `clear_tv(rettv)` before the `rettv_blob_set(rettv, new_blob)` is no longer needed.